utils/pytest.ini

```diff
@@ -1,3 +0,0 @@
-[pytest]
```

Comment:
Why is this file being removed?

Reply:
Its scope was limited and the purpose is redundant.

Reply:
It looks like it has lost the `norecursedirs = vlog` setting? Is that no longer needed...

Reply:
Correct. It seems to do fine on everything so whatever required it in the [commit](https://github.com/SymbiFlow/symbiflow-arch-defs/commit/e6c4952ce7c2c8278aa57e01d71c8bedf48c97a3) seems to be resolved.

Reply:
Great!